### ISSUE TITLE
Missing Nokogiri dependency?

### ISSUE THREAD

**Issue by DieterDePaepe** [nanoc/nanoc#1053]
On a fresh (windows) installation, nanoc fails to start due to a failure to load Nokogiri.

### Steps to reproduce

Follow steps decribed on https://nanoc.ws/doc/installation/
On commandline: `nanoc help`

### Expected behavior

A help printout.

### Actual behavior

A stack trace - see below

### Details

OS: Windows 7
Ruby: ruby 2.2.6p396 (2016-11-15 revision 56800) [i386-mingw32]
Nanoc: 
```
Nanoc 4.5.0 (c) 2007-2017 Denis Defreyne.
Running ruby 2.2.6 (2016-11-15) on i386-mingw32 with RubyGems 2.4.5.2.
```

Resolved after doing `gem install nokogiri`.

### Crash log

[Gist](https://gist.github.com/DieterDePaepe/7bd0567f915b96cb18f8bae9738c936f)

**Comment by ddfreyne** [comment]
This is indeed wrong. The problem is not a missing nokogiri dependency, but rather that Nokogiri should be required when needed, rather than eagerly. Fix coming up.

**Comment by ddfreyne** [comment]
This bug is sadly tricky to test for automatically.

**Comment by ddfreyne** [comment]
Fixed in Nanoc 4.5.1.
